src/frontend/src/handler/create_source.rs

```diff
@@ -1366,6 +1366,7 @@ pub async fn extract_iceberg_columns(
                 }
             })
             .collect();
+        columns.push(ColumnCatalog::iceberg_sequence_num_column());
```

Comment:
Let's document this additional field in the schema of `LogicalIcebergScan` and the `IcebergScanExecutor`.